kubernetes/cray-kafka-operator/values.yaml

```diff
@@ -7,78 +7,92 @@ strimzi-kafka-operator:
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   kafkaConnects2i:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   topicOperator:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: operator
-      tag: 0.15.0
+      tag: 0.27.0
   userOperator:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: operator
-      tag: 0.15.0
+      tag: 0.27.0
   kafkaInit:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: operator
-      tag: 0.15.0
+      tag: 0.27.0
   tlsSidecarZookeeper:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   tlsSidecarKafka:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   tlsSidecarEntityOperator:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   kafkaMirrorMaker:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
   kafkaBridge:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka-bridge
-      tag: 0.15.0
+      tag: 0.21.2
   kafkaExporter:
     image:
-      repository: artifactory.algol60.net/csm-docker/stable/docker.io/strimzi
+      registry: artifactory.algol60.net
+      repository: csm-docker/stable/quay.io/strimzi
       name: kafka
-      tagPrefix: 0.15.0
+      tagPrefix: 0.27.0
 
-# Increase resources.limits.cpu to 8 to reduce cpu throttling
+  # Increase resources.limits.cpu to 8 to reduce cpu throttling
```

Comment:
Was this indentation intentional or accidental?

Reply:
My editor indented it to match the indention of the code below